include/irr/asset/ICPUDescriptorSet.h

```diff
@@ -128,7 +128,10 @@ class ICPUDescriptorSet final : public IDescriptorSet<ICPUDescriptorSetLayout>,
 		_IRR_STATIC_INLINE_CONSTEXPR auto AssetType = ET_DESCRIPTOR_SET;
```

Comment:
convertToDummyObject above this line does not check for mutability

Reply:
because it doesnt have to, it would change nothing